File: AtCoder/ABC/174/python/D_Alter_Altar.py
N = int(input())
C = input()
cnt_W = 0
cnt_R = 0
for c in C:
    if c == 'W':
        cnt_W += 1
    else:
        cnt_R += 1
res = 0
for c in C[:cnt_R]:
    if c == 'W':
        res += 1
print(res)

File: AtCoder/ABC/174/cpp/D_Alter_Altar.cpp
#include <bits/stdc++.h>
using namespace std;

int N;
string C;

void solve() {
  int cnt_R = 0;
  int cnt_W = 0;
  for (auto c : C) {
    if (c == 'R') {
      ++cnt_R;
    } else {
      ++cnt_W;
    }
  }
  int res = 0;
  for (int i = 0; i < cnt_R; ++i) {
    if (C[i] == 'W') {
      ++res;
    }
    printf("%d\n", res);
  }

  int main() {
    cin >> N;
    cin >> C;
    solve();
  }